evalml/pipelines/components/transformers/preprocessing/text_featurizer.py

```diff
@@ -101,11 +100,11 @@ def transform(self, X, y=None):
         if self._features is None or len(self._features) == 0:
             return X
 
-        text_columns = self._get_text_columns(X)
-        es = self._make_entity_set(X, text_columns)
+        es = self._make_entity_set(X, self._text_columns)
```

Comment:
In `_make_entity_set`, I think we need to switch from "text" to "natural_language" because "text" is now deprecated:

![image](https://user-images.githubusercontent.com/41651716/103825701-5be99e00-5043-11eb-83b5-1f8121cb11ea.png)